android/src/main/java/com/swmansion/reanimated/nodes/AlwaysNode.java

```diff
@@ -1,12 +1,13 @@
 package com.swmansion.reanimated.nodes;
 
 import com.facebook.react.bridge.ReadableMap;
+import com.swmansion.reanimated.MapUtils;
 import com.swmansion.reanimated.NodesManager;
 
 public class AlwaysNode extends Node implements FinalNode {
   public AlwaysNode(int nodeID, ReadableMap config, NodesManager nodesManager) {
     super(nodeID, config, nodesManager);
-    mNodeToBeEvaluated = config.getInt("what");
+    mNodeToBeEvaluated = MapUtils.getInt(config, "what", "Reanimated: Argument passed to always node is either of wrong type or is missing. NodeID: " + nodeID);
```

Comment:
I'd prefer to avoid doing it this way. We add error messages at a cost of additional string concatenation and few additional allocations. Note that before `getInt` method is called we need to know the value of the string which requires JVM to autobox nodeID and concat with a string even though it is unlikely that an error would happen (especially in production these errors aren't worth much)

I'd prefer if we just add a generic error (plain string in this case) that would at least allow to trace the error back to the place where it's been thrown. Also not sure if having node ID in the error message is helpful at all

Reply:
I will get to that soon 🙂

Reply:
Apologies, It took so long. I deleted the node ID. Your explanation makes perfect sense. Let me know what you think right now :)